vlasovsolver/cpu_trans_map_amr.cpp

```diff
@@ -70,6 +75,26 @@ int getNeighborhood(const uint dimension, const uint stencil) {
    
 }
 
+bool check_is_local(const dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid, CellID cell, int dimension) {
+   if (P::vlasovSolverLocalTranslate) {
+      switch (dimension) {
+         case 0:
+            if (LocalSet_x.count(cell)) return true;
+            break;
+         case 1:
+            if (LocalSet_y.count(cell)) return true;
+            break;
+         case 2:
+            if (LocalSet_z.count(cell)) return true;
+            break;
+      }
+      return false;
+   } else {
+      if (mpiGrid.is_local(cell)) return true;
+      return false;
```

Comment:
General style bickering... Please add a default case just to avoid nasty surprises the day someone sends a 3 for dimension, and use { } even in single-line ifs as someone is bound to add a second line and forget one day...